native/sapp-wasm/src/lib.rs

```diff
@@ -280,17 +280,56 @@ extern "C" {
     pub fn init_opengl();
     pub fn canvas_width() -> i32;
     pub fn canvas_height() -> i32;
-    pub fn test_log(msg: *const ::std::os::raw::c_char);
+    pub fn raw_console_debug(msg: *const ::std::os::raw::c_char);
+    pub fn raw_console_log(msg: *const ::std::os::raw::c_char);
+    pub fn raw_console_info(msg: *const ::std::os::raw::c_char);
+    pub fn raw_console_warn(msg: *const ::std::os::raw::c_char);
+    pub fn raw_console_error(msg: *const ::std::os::raw::c_char);
+}
+
+pub fn console_debug(msg: &str) {
```

Comment:
I would prefer to keep `sapp_wasm` as low level as possible and build higher level abstractions in other crates. 
Just `fn console_debug(msg: *_)` looks good enough for `sapp_wasm`. 

I think your logging crate is a better place for `console_log(msg: &str)`

Reply:
Oh, I didn't realize `raw_console_*` is available as a public interface. Will change.